ftp/src/test/scala/akka/stream/alpakka/ftp/CommonFtpStageSpec.scala

```diff
@@ -368,4 +370,36 @@ trait CommonFtpStageSpec extends BaseSpec with Eventually {
       extraWaitForStageShutdown()
     }
   }
+
+  "FtpDirectoryOperationsSource" should {
+    "make a directory in current path" in {
+      val basePath = "/upload"
+      val name = "sample_dir_" + Instant.now().getNano
+
+      val source = mkdir(basePath, name)
```

Comment:
IIRC the testcases are running against the `./ftp/tmp/home` directory. So you can check in the test if the newly created directory exists.

https://github.com/akka/alpakka/blob/7dc6727bd2e02e562f0097f316017f2754211a0e/docker-compose.yml#L75

Reply:
Yes, they are. 
I will update those tests